deployments/local-minio/deploy.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+# Name to give the docker network and stack. Update network instances in docker-compose.yml file if modified.
+networkname="cdr_local-minio"
+
+function wait_for_containers () {
+    waiting=$(docker service ls | grep ""$networkname"_.*0/1 ")
+    printf "\nWaiting for docker services to start."
+    while [ ! -z "$waiting" ]; do
+        sleep 2
+        printf "."
+        waiting=$(docker service ls | grep ""$networkname"_.*0/1 ")
+    done
+    printf "\nDone!\n"
+}
+
+# Deploy docker-compose.yml file via docker stacks
+function deploy_images () {
+    printf "Deploying Docker Images...\n"
+    docker stack deploy -c docker-compose.yml $networkname
+}
+
+# Create attachable overlay docker network
+function create_network () {
+    printf "Creating docker '$networkname' network...\n"
+    docker network create --driver=overlay --attachable $networkname
+}
+
+# Check is network already does not exist or is not attachable.
+# Prompt to create if nonexistent or continue if network exists and is attachable.
+function check_network_configuration () {
+    printf "Checking Docker Network...\n"
+    network=`docker network ls | grep "$networkname"`
+    networktype=$(docker network inspect $networkname 2>/dev/null | grep -i "\"driver\": \"overlay\"")
+    attachable=$(docker network inspect $networkname 2>/dev/null | grep -i "\"attachable\": true")
+    if [ -z "$network" ]; then
+        read -p "Network '$networkname' does not exist, create it now? (y/N) " -n 1 -r
```

Comment:
If the network doesn't exist, go ahead and create it. Print a message that a new network is being create. Do not prompt the user for input.

Reply:
I put a confirmation here because creating an external docker network on the user's machine seems a step outside of what a user would expect a deploy script to do - just bring up/down a docker-compose file.